composer.json

```diff
@@ -6,17 +6,18 @@
     "require": {
         "php": ">=7.4",
         "ext-json": "*",
+        "guzzlehttp/oauth-subscriber": "^0.4",
+        "keboola/filter": "^2.0",
         "keboola/juicer": "dev-webrouse-COM-596-update-guzzle",
-        "keboola/php-utils": "^4.1",
         "keboola/php-csvtable": "^1.2",
-        "keboola/filter": "^2.0",
+        "keboola/php-temp": "^2.0",
+        "keboola/php-utils": "^4.1",
+        "kevinrob/guzzle-cache-middleware": "^3.3",
```

Comment:
Generic extractor podporuje cache.
Nie je to verejne zdokumentovana vlastnost, pouziva sa iba pri vyvoji sablon, vid: 
https://github.com/keboola/generic-extractor/issues/62